packages/nimble-components/src/button/types.ts

```diff
@@ -4,16 +4,8 @@
  */
 import type { Button } from '@microsoft/fast-foundation';
 
-export { ButtonAppearance } from '../patterns/button/types';
+export {
```

Comment:
Scanning how we do it for other types that are reused across components, I think we have a pattern where each component will re-export the types. You're doing that here for the button but it's missing for the anchor button.

Reply:
Whoops. Fixed.